### PR TITLE
List DaiEth,RicUsdc,MaticUsd , WbtcDai and MaticDai two way markets

### LINKED ISSUE #558: List USDC<>RIC Market
This issue is to list USDC<>RIC market in the app, the contract address is: 0xFBF91D299db56624f46F544dc1DCD0d0da2e3327

Also, since this market uses RIC as the pair token, the subsidy token is ETHx, I believe for things to work correctly this needs to be set in the flowConfig

### LINKED ISSUE #559: List DAI<>ETH Market
The DAI<>ETH Market contract is: 0x47F99Bf24C7f91Bc93bF3b7f0b3D65eef1583dF6

### LINKED ISSUE #570: List MATIC<>USDC
The MATIC<>USDC contract address is: 0x7688f3efA3A19948b07c26B5B72F67a42d102C83, I'm keeping it at the top of the hour, so txns to open streams might revert since the oracle isn't updated that frequently

### LINKED ISSUE #572: List MATIC<>DAI
This is the MATIC<>DAI contract: 0xbb131E3Cd2FadE7f387c2024F04F33Ed5e6c5101

### LINKED ISSUE #575: List WBTC<>DAI
List the WBTC<>DAI market: 0x2cAD92C88fc424779AB53e17A948217A3Be83559

### PR DESCRIPTION
**Pull request by elvijsTDL**
Fixes #559, #558, #570 , #572 , #575

### CODE REVIEW

**Comment by elvijsTDL** [comment]
Distributions show up 52 years ago because there have never been one yet, will leave the streams open , but going off for now
Screenshot 2022-03-05 at 02 10 38

File: src/constants/flowConfig.ts [hunk 1]
```diff
@@ -60,6 +67,38 @@ export const indexIDA: IndexIDAType = [
...
+    outputIndex: 0,
+  },
+  {
+    input: USDCxAddress,
+    output: RICAddress,
+    subsidy: WETHxAddress,
+    subsidyIndex: 2,
+    inputIndex: 0,
+    outputIndex: 1,
+  },
```
Comment (mikeghen) [inline src/constants/flowConfig.ts:119]: I think these might be off, I tried to open a stream USDC>>RIC and got this error:
Screen Shot 2022-03-04 at 7 31 57 PM

Comment (mikeghen) [reply src/constants/flowConfig.ts:119]: https://dashboard.tenderly.co/tx/polygon/0x1fc6c3f5b048ecf52159e45a570d5661fa366c8ea0064f2391a443e09f59032a

**Review by mikeghen — APPROVED** [review]

**Review by mikeghen — CHANGES_REQUESTED** [review]
I think the index might be messed up on the USDC>>RIC side of the exchange

**Review by mikeghen — CHANGES_REQUESTED** [review]
Can you change the ETH<>DAI market to this address: 0xEAA1d76D0aB8d0cab426e7427c472C638cFc5EA1

The previous one had an issue with the lastDistributedAt

**Comment by mikeghen** [comment]
@elvijsTDL, let me know when you get a chance to review the requested changes

**Review by mikeghen — CHANGES_REQUESTED** [review]
@elvijsTDL, I just deployed the MATIC<>USDC market: 0x7688f3efA3A19948b07c26B5B72F67a42d102C83, could you add this too?

**Comment by mikeghen** [comment]
@elvijsTDL also added another $500 bounty for listing MATIC<>USDC :+1: so this PR with all the markets would be a $1500 bounty 
Screen Shot 2022-03-09 at 10 03 22 AM

**Comment by mikeghen** [comment]
@elvijsTDL the MATIC<>DAI contract address: 0xbb131E3Cd2FadE7f387c2024F04F33Ed5e6c5101

**Review by mikeghen — APPROVED** [review]
Well done @elvijsTDL 
Screen Shot 2022-03-10 at 11 50 25 AM
Screen Shot 2022-03-10 at 11 50 00 AM

**Pull request merged**